server/src/test/java/io/envoyproxy/controlplane/server/EnvoyContainer.java

```diff
@@ -24,7 +24,7 @@
   private final Supplier<Integer> controlPlanePortSupplier;
 
   EnvoyContainer(String config, Supplier<Integer> controlPlanePortSupplier) {
-    super("envoyproxy/envoy-alpine-dev:latest");
+    super("envoyproxy/envoy-alpine-dev:8fb3cb86082b17144a80402f5367ae65f06083bd");
```

Comment:
@wookieJ - please add that this needs to be updated here: https://github.com/envoyproxy/java-control-plane#bring-api-up-to-date-with-data-plane-api. Also @nezdolik I think that this is a better way to have repeatable builds and solves this https://github.com/envoyproxy/java-control-plane/issues/154 . In the next release of java-control-plane we will remove v2 usage and update to to 1.16.1.

Reply:
no, manual changes only to the files you specified. And `README.md` soon

Reply:
added

Reply:
nice! lmk if I could help with deprecating v2